### DISCUSSION TITLE
[WASM] ToastContent workaround

### DISCUSSION THREAD

**Discussion by SanAlexis** [unoplatform/uno#11614] [Q&A] [answered]
I'm doing a migration project from a project targeting UWP `uap10.0.17134.0` to a project targeting WASM `net6.0`. I'm looking for a workaround to use a notification mecanism like `ToastContent` which could be use only for `WINDOWS_UWP`.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. 

The `ToastContent` API is not supported by Uno at this time, at least in in a cross platform way. You can use [platform specific code](https://platform.uno/docs/articles/platform-specific-csharp.html) to use it on the `.UWP` head, and you can use native APIs directly for the other platforms to achieve similar results (if the underlying platform supports it).
